### PR TITLE
Reality check validation update

### PR DESCRIPTION
**Pull request by gokula-krishna-dev**

### CODE REVIEW

File: src/botPage/view/View.js [hunk 1]
```diff
@@ -337,14 +337,21 @@ export default class View {
...
             }
         });
 
+        $('#realityDuration').keypress(e => {
+            const char = String.fromCharCode(e.which);
+            if ((!/[0-9]/.test(char) && [8, 37, 39].indexOf(e.keyCode) < 0) || /['%]/.test(char)) {
+                e.preventDefault();
+            }
+        });
+
```
Comment (gokula-krishna-dev) [inline src/botPage/view/View.js]: QA requested with binary static behaviour.

File: src/botPage/view/View.js [hunk 2]
```diff
@@ -337,14 +337,21 @@ export default class View {
...
                 hideRealityCheck();
                 startRealityCheck(time);
             } else {
                 $('#rc-err').show();
             }
         });
 
+        $('#realityDuration').keypress(e => {
+            const char = String.fromCharCode(e.which);
+            if ((!/[0-9]/.test(char) && [8, 37, 39].indexOf(e.keyCode) < 0) || /['%]/.test(char)) {
```
Comment (aminmarashi) [inline src/botPage/view/View.js]: Better to write it this way:

```
if (/[0-9]/.test(char)) return;
e.preventDefault();
```

File: yarn.lock [hunk 3]
```diff
@@ -72,10 +72,6 @@
     normalize-path "^2.0.1"
     through2 "^2.0.3"
 
-"@types/node@*":
```
Comment (aminmarashi) [inline yarn.lock]: @gokula-krishna-dev Could you please remove the changes in `yarn.lock`?

File: package.json [hunk 4]
```diff
@@ -115,7 +115,7 @@
     "gulp-sourcemaps": "^2.6.2",
     "gulp-watch": "^4.3.11",
     "husky": "^0.14.3",
-    "i18next-scanner": "^2.2.1",
```
Comment (aminmarashi) [inline package.json]: @gokula-krishna-dev This change also needs to be removed.

**Pull request merged**